system/Test/FeatureResponse.php

```diff
@@ -98,6 +98,27 @@ public function assertRedirect()
 		$this->assertTrue($this->isRedirect(), 'Response is not a redirect or RedirectResponse.');
 	}
 
+	/**
+	 * Assert that a given response was a redirect
+	 * and it was redirect to a specific URI.
+	 *
+	 * @param string $uri
+	 */
+	public function assertRedirectedTo(string $uri)
+	{
+		if (! $this->isRedirect())
+		{
+			return false;
+		}
+
+		$uri         = trim(strtolower($uri));
```

Comment:
Why lower case? I don't think the return from `getRedirectUrl()` will necessarily be lower.

Reply:
might be true. I know headers are case-insensitive, which obviously isn't the same. Will turn both to lowercase then.